kuma/static/js/experiment-static-examples-on-top.js

```diff
@@ -0,0 +1,14 @@
+(function() {
```

Comment:
I have no idea what you are saying, but I will do it.  Does [experiment-framework-test.js](https://github.com/mozilla/kuma/blob/master/kuma/static/js/experiment-framework-test.js) and [experiment-wiki-content.js](https://github.com/mozilla/kuma/blob/master/kuma/static/js/experiment-wiki-content.js) need it too?